<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientInfoForm</class>
 <widget class="QWidget" name="PatientInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="genderLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>성별</string>
   </property>
  </widget>
  <widget class="QLabel" name="treatLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>치료</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>641</width>
     <height>341</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>나이</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>성별</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>진단</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>치료</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="genderComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="treatComboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="diagnosisLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>진단</string>
   </property>
  </widget>
  <widget class="QComboBox" name="diagnosisComboBox">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>60</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
